api/_globals/reportError.json

```
@@ -0,0 +1,113 @@
+{
+  "api": {
+    "reportError": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/reportError",
+        "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#dom-reporterror",
```

It looks like there's a non-normative block on this spec page, can we link to that instead?

```suggestion
        "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#runtime-script-errors",
```